docs/design.rst

```diff
@@ -1,20 +1,28 @@
-ToDD - High-Level Design
+High-Level Design of ToDD
 ================================
 
-The High-Level Design of ToDD is fairly simple:
+The High-Level Design of ToDD is fairly simple. ToDD is composed of three components:
+
+* Server
+* Agent
+* Client
+
+A general idea of how these components is depicted below:
 
 .. image:: images/todd-hld.png
 
 Some notes about this:
 
-* All database integrations are at the server level
-* Agents communicate with server via "comms" abstraction (currently RMQ is supported)
-* Server provides REST API to either the pre-packaged "todd" client tool, or to 3rd party software
+* All database integrations are at the server level - no agent communicates with database
+* Agents _do not_ communicate directly with server. This is done through some kind of message queue (i.e. RabbitMQ) using ToDD's "comms" abstraction.
```

Comment:
Fixed in https://github.com/Mierdin/todd/pull/94/commits/7912f3dab6a00105edf86dd071af2cfad21bbe4c